libc/docs/dev/code_style.rst

```diff
@@ -186,3 +186,32 @@ We expect contributions to be free of warnings from the `minimum supported
 compiler versions`__ (and newer).
 
 .. __: https://libc.llvm.org/compiler_support.html#minimum-supported-versions
+
+Policy on Assembler sources
```

Comment:
Pet peeve: assembly (language), not assembler (just because the latter is in the dictionary, used by some people and what -x accepts doesn't mean it's a sensible name for it; assembler is really the tool not the language)

Reply:
257fbfaeb0d3